charts/business-central/templates/service.yaml

```diff
@@ -7,6 +7,12 @@ metadata:
 spec:
   type: {{ .Values.service.type }}
   ports:
+    {{- if .Values.service.ports.ssh }}
```

Comment:
I think we don't need that here, that would be just needed if other containers would need to access SSH